### PR TITLE
Update style.css

### PR DESCRIPTION
**Pull request by AurelioDeRosa**
Fix issue jquery/jquery.org#36

### CODE REVIEW

File: themes/jquery.org/style.css [hunk 1]
```diff
@@ -612,6 +612,14 @@ nav#main ul {
...
 		text-align: center;
 	}
+
+	nav#main .searchform {
+		width: 100%;
+	}
+
+	nav#main ul {
+		width: 100%;
+	}
```
Comment (arthurvr) [inline themes/jquery.org/style.css]: Why not to combine this one with the previous block?

Comment (AurelioDeRosa) [inline themes/jquery.org/style.css]: Which previous block? Do you mean the very similar media query (1px diff) after this one?

Comment (arthurvr) [inline themes/jquery.org/style.css]: like

```css
nav#main .searchform,
nav#main ul {
        width: 100%;
}
```

Comment (arthurvr) [inline themes/jquery.org/style.css]: ping @AurelioDeRosa

**Comment by AurelioDeRosa** [comment]
@arthurvr Done.

**Pull request closed without merging**